client/js/Ui/core.js

```diff
@@ -32,6 +32,10 @@ SCWeb.ui.Core = {
               self.resolveElementsAddr('body'),
             ]).then(function () {
 
+                document.getElementById('header').style.display = "block";
```

Comment:
Please add comment before this lines like "Display interface elements only after page load"